plugin/evm/validators_manager_test.go

```diff
@@ -0,0 +1 @@
+package evm
```

Comment:
?

Reply:
Yes, this is the testing file that was left from some previous refactoring and that most probably will be used again.
Deleted for now